src/routes/application.js

```diff
@@ -7,15 +7,32 @@ module.exports = class extends Route {
 		super(...args, { route: 'application' });
 	}
 
-	get(request, response) {
+	async get(request, response) {
+		let [users, guilds, channels, memory] = [0, 0, 0, 0];
+
+		if (this.client.shard) {
```

Comment:
shouldn't this be `if (!this.client.shard)` because the else block is doing the broadcastEval.